Manito/Manito/Screens/Login/View/LoginView.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  LoginView.swift
+//  Manito
+//
+//  Created by COBY_PRO on 10/24/23.
+//
+
+import AuthenticationServices
+import Combine
+import UIKit
+
+import SnapKit
+
+final class LoginView: UIView, BaseViewType {
+    
+    // MARK: - ui component
+
+    private let logoImageView: UIImageView = UIImageView(image: UIImage.Image.appIcon)
+    private let logoTextImageView: UIImageView = UIImageView(image: UIImage.Image.textLogo)
+    private let appleLoginButton: ASAuthorizationAppleIDButton = {
+        let button = ASAuthorizationAppleIDButton(type: .signIn, style: .white)
+        button.cornerRadius = 25
+        return button
+    }()
+    
+    // MARK: - property
+    
+    let appleSignButtonDidTapPublisher = PassthroughSubject<Void, Never>()
```

Comment:
해당 Subject를 따로 만들지 않고 기존의 버튼 TapPublisher처럼 버튼에 이벤트를 발생시킬 수 있습니다.
하지만 현재 appleLoginButton이 UIButton이 아니라 UIControl을 상속받기 때문에 기존의 tapPublisher를 사용할 수 없더라구요

그래서 따로 extension을 만들어두고 사용할 수 있습니다.
```swift
//  UIControl+Combine.swift

extension UIControl {
    var buttonTapPublisher: AnyPublisher<Void, Never> {
        self.controlPublisher(for: .touchUpInside)
            .map { _ in Void() }
            .eraseToAnyPublisher()
    }
}
```

```suggestion
    var appleSignButtonTapPublisher: AnyPublisher<Void, Never> {
        return self.appleLoginButton.buttonTapPublisher
    }
```

이렇게 수정한다면 아래의 
```swift

    // MARK: - func

    private func setupAction() {
        let action = UIAction { [weak self] _ in
            self?.appleSignButtonDidTapPublisher.send()
        }
        self.appleLoginButton.addAction(action, for: .touchUpInside)
    }
```

해당 부분도 다 지울 수 있습니다.

UIControl Extension의 이름을 `buttonTapPublisher`로 한 이유는 tapPublisher로 했을 때
<img width="630" alt="image" src="https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/assets/78677571/6d4df630-3313-4570-8d28-d36bcb7ebf03">
위 사진과 같은 오버라이드 에러가 발생하기 때문입니다.

Reply:
[72702e3](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/580/commits/72702e38895d08de9ec4c0ed865f26019351510d)
감사합니다..케미..! 수정하였습니다.